tests/functional/lib/helpers.js

```diff
@@ -180,6 +180,28 @@ define([
     }, [ selector ], timeout);
   }
 
+  function notVisible(context, selector) {
```

Comment:
not sure about this helper, 

https://theintern.github.io/leadfoot/Element.html#isDisplayed

for something `notVisible` there is `isDisplayed` that also has a 0 timeout if you just use it on its own.
If we are looking for something that is not on the page then can't we just use `setFindTimeout(some_timeout_to_wait_for_error_to_appear)` + `findBy...` ?

This might just need renaming so we do not confuse it with `isDisplayed()`